lib/pfconfig/namespaces/config/Pf.pm

```diff
@@ -115,8 +115,10 @@ sub build_child {
             $Config{$group}{$item} = File::Spec->catfile( $log_dir, $Config{$group}{$item} );
         }
     }
-
-    $Config{trapping}{passthroughs} = [ split( /\s*,\s*/, $Config{trapping}{passthroughs} || '' ) ];
+    foreach my $val ("trapping.passthroughs") {
```

Comment:
This was a refactor done because of the extended namespace.
However we removed that in a later commit.